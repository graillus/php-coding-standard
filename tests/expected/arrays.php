<?php

declare(strict_types=1);

$arraySyntax = [1, 2];

[$list, $syntax] = [1, 2];

$multilineWithSingleElement = ['foo'];

$multilineWithoutTrailingComma = [
    'foo',
    'bar',
];

$singleLineWithMultipleElements = ['one', 'two', 'three'];
