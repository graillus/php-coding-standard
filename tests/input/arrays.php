<?php

declare(strict_types=1);

$arraySyntax = array(1, 2);

list($list, $syntax) = [1, 2];

$multilineWithSingleElement = [
    'foo',
];

$multilineWithoutTrailingComma = [
    'foo',
    'bar'
];

$singleLineWithMultipleElements = ['one', 'two', 'three'];
