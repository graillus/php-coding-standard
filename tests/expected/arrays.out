
FILE: tests/input/arrays.php
------------------------------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 4 LINES
------------------------------------------------------------------------------------------
  5 | ERROR | [x] Short array syntax must be used to define arrays
  7 | ERROR | [x] list(...) is forbidden, use [...] instead.
  9 | ERROR | [x] Multi-line array contains a single value; use single-line array instead
 15 | ERROR | [x] Multi-line arrays must have a trailing comma after the last element.
------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------


